src/main.cc

```diff
@@ -656,6 +656,7 @@ print_help(FILE *stream, const char *program)
 		program, program, program, program);
 }
 
+#if !defined(ENABLE_FAILOVER)
```

Comment:
This looks a little strange. What I mean is that we only use this function if `ENABLE_FAILOVER` is 1, and this function exists only in this case. How about inlining this function or removing the `static` from it along with `#if !defined(ENABLE_FAILOVER)`?

Reply:
OK. Added the `fprintf()` call directly into the switch-case.